api/healthCheck/index.ts

```diff
@@ -47,26 +44,27 @@ export function getPromises(path): any[] {
             break
         }
     }
-
-    const health = getConfigValue(HEALTH)
-
     if (healthCheckEndpointDictionary[path]) {
+        console.log(healthCheckEndpointDictionary[path])
```

Comment:
`console.log()` to go, unless we really need it.

Reply:
removed this line as its not necessary